aws/resource_aws_codebuild_project.go

```diff
@@ -197,6 +197,26 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 							Optional:     true,
 							ValidateFunc: validation.StringMatch(regexp.MustCompile(`\.(pem|zip)$`), "must end in .pem or .zip"),
 						},
+						"registry_credential": {
+							Type:     schema.TypeSet,
```

Comment:
When using `MaxItems: 1` for configuration block attributes, we prefer to simply the attribute type to `TypeList` instead. 👍 

```suggestion
							Type:     schema.TypeList,
```